### ISSUE TITLE
problem to install haskintex (fail to build)

### ISSUE THREAD

**Issue by stla** [Daniel-Diaz/haskintex#5]
Hello, 
I've run
`cabal install haskintex`
The installation was successful for all the dependencies (installed in the sandbox).
But there's a failure for haskintex:
```
Configuring haskintex-0.6.0.1...
Building haskintex-0.6.0.1...
Failed to install haskintex-0.6.0.1
Build log ( /home/stla/.cabal-sandbox/logs/haskintex-0.6.0.1.log ):
Configuring haskintex-0.6.0.1...
Building haskintex-0.6.0.1...
Preprocessing library haskintex-0.6.0.1...
[1 of 2] Compiling Paths_haskintex  ( dist/dist-sandbox-905ee030/build/autogen/Paths_haskintex.hs, dist/dist-sandbox-905ee030/build/Paths_haskintex.o )
[2 of 2] Compiling Haskintex        ( Haskintex.hs, dist/dist-sandbox-905ee030/build/Haskintex.o )

Haskintex.hs:188:16:
    Expecting one more argument to ‘H.Exp’
    Expected a type, but ‘H.Exp’ has kind ‘* -> *’
    In the type signature for ‘processExp’:
      processExp :: (H.Exp -> H.Exp) -> Text -> Parser Text
cabal: Error: some packages failed to install:
haskintex-0.6.0.1 failed during the building phase. The exception was:
ExitFailure 1
```

**Comment by Daniel-Diaz** [comment]
Thanks for the report.

This is due to some breaking changes from `haskell-src-exts`. I'm working on a fix.

**Comment by Daniel-Diaz** [comment]
Fixed in `haskintex-0.7.0.0`.

**Comment by stla** [comment]
It works. Thank you.
